File: test/fixtures/node-got/post_same_field_multiple_times.js
import got from 'got';

const response = await got.post('http://localhost:28139/', {
  headers: {
    'Content-Type': 'application/x-www-form-urlencoded'
  },
  body: new URLSearchParams([
    ['foo', 'bar'],
    ['foo', ''],
    ['foo', 'barbar']
  ]).toString()
});

File: test/fixtures/ruby-httparty/post_same_field_multiple_times.rb
require 'httparty'

url = 'http://localhost:28139/'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = {
  'foo' => [
    'bar',
    '',
    'barbar'
  ]
}
res = HTTParty.post(url, headers: headers, body: body)
